src/helpers/tokenHelpers/projects/BLONKShelpers.ts

```diff
@@ -0,0 +1,94 @@
+import { IToken } from '../../../db/schemas/schemaTypes';
+import { ProcessManyMintsFunction } from '../types';
+import { allBLONKStraits } from '../../constants';
+// import { fetchSVG } from '../../../web3/blockchainFetches';
+import { addManyTokens } from '../../../db/queries/tokenQueries';
+import {
+  updateProjectSupplyAndCount,
+  getProjectCurrentSupply,
+} from '../../../db/queries/projectQueries';
+
+export const processBlonksMint: ProcessManyMintsFunction = async (
+  token_ids,
+  project,
+  contract,
+  context,
+  conn,
+) => {
+  const {
+    _id: project_id,
+    project_name,
+    project_slug,
+    artist,
+    artist_address,
+    description,
+    collection_name,
+    script_type,
+    aspect_ratio,
+    website,
+    external_url,
+    license,
+    royalty_info,
+    tx_count,
+  } = project;
+
+  const newTokens: IToken[] = [];
+
+  context.log.info(`Beginning extraction of ${project_name} svgs from the blockchain`);
+
+  for await (const tokenId of token_ids) {
+    // TODO Make unique types for tokens and projects.
+    const newToken: IToken = {
+      token_id: tokenId,
+      // FIXME token name is hardcoded for blonks rn. Abstract out name generation of a token somehow.
+      name: `BLONK #${tokenId + 1}`,
```

Comment:
This is actually pretty standard to do it this way. But we will probably leave it zero-indexed. And make sure you add the 'S', BLONKS*.

Reply:
Cool. Will do, Matto actually told me to name it with a singular